task3_1/index.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE>
```

Comment:
looks like you made it in another PR
if yes, please remove these files from here, thanks